isovar/dataframe_builder.py

```diff
@@ -133,6 +146,9 @@ def add(self, variant, element):
                 name = self.rename_dict[name]
             self.columns_dict[name].append(value)
 
+        for column_name, fn in self.extra_column_fns.items():
+            self.columns_dict[column_name] = fn(variant, element)
```

Comment:
I agree with this, but not sure what the `fn` would be in this case? Could also just add another flag to the function for a `gene` columns, but that starts to get messy.